### DISCUSSION TITLE
Option to disable dual pane and only have a single one

### DISCUSSION THREAD

**Discussion by jay-ramani** [doublecmd/doublecmd#1948] [Suggestions]
Would be useful to have an option to disable dual panes and have a single pane. Many other file managers like NexusFile, FreeCommander, MultiCommander, etc. have this option.

**Comment by alexx2000** [comment]
This will never happen. DC is a two-panel file manager, this is a fundamental principle.

**Comment by ags1234** [comment]
@jay-ramani, there is a sub-type of file managers that is called orthodox file managers and these have dual panes.
We are all gathered here because we like to have dual panes, we find that workflow more productive.
DC is an orthodox file manager.

**Reply by jay-ramani** [reply]
I'm not asking to convert the file manager to a single pane one. Just asking if there could be an option to have a single pane.

**Reply by JonnyTech** [reply]
In case you missed it, from the author of DC:

> This will never happen.

**Comment by alexx2000** [comment]
@jay-ramani Technically you can hide one panel (set width to zero) in DC 1.2. But I guess it is not that you want. DC still will work as dual-panel file manager in this case.

**Comment by Skif-off** [comment]
Maybe something like [this](https://doublecmd.h1n.ru/viewtopic.php?t=6521) can help?

**Reply by antodc** [reply]
The lua script is really very nice! I think it is handy to have the possibility to use only one panel, especially when working with a small screen.
@Skif-off Do you mind if I ask you to modify the script to disable the two drive bars when using one panel and enable them with two panels? If feasible and not difficult, of course. Please! :-)

**Reply by Skif-off** [reply]
> disable the two drive bars when using one panel and enable them with two panels

Lua does not have access to this parameter yet.

**Comment by savchenko** [comment]
This would be indeed desirable for situations when screen space is limited and/or user wants to focus on a single panel.
